R/summarise_country.R

```diff
@@ -0,0 +1,163 @@
+summarise_sheet_indicator_list <- function(billion) {
+  billion <- rlang::arg_match(billion)
+  indicator_order_b <- indicator_order %>%
+    dplyr::filter(ind %in% billion_ind_codes(billion)) %>%
+    dplyr::arrange(order) %>%
+    dplyr::select(sdg, short_name, medium_name, unit_raw, )
+
+  return(indicator_order_b)
+}
+
+#' Summarise Excel sheet 'data' for HPOP country summary
+#'
+#' `summarise_HPOP_cntry_data` summarises the 'data' sgeet for the HPOP country summary.
+#'
+#' @param iso3 ISO3 codes of country to export.
+#' @param ... Additional arguments passed to `calculate_hpop_contributions` and
+#' `calculate_hpop_billion`. Use if you need to provide additional parameters to
+#' those functions.
+#' @inherit calculate_hpop_contributions params
+#'
+#' @return list of data frames to be used in the 'data' sheet of HPOP country summary
+#' export
+#'
+#' @export
```

Comment:
Does this need to be exported? Could this just be internal to the Excel generation function or think there's use for now of having exported? I guess if useful otherwise we could export but my intuition would be not to unless necessary.

Reply:
I agree, I don't think it's going to be used for other purposes.